odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/utils/JSONSerDeUtils.java

```diff
@@ -43,6 +45,14 @@
         }
     }
 
+    public static <T> T deserializeJson(final File file, final TypeReference<T> tr) {
```

Comment:
please create Util class in test sources, since this method will not be used outside of tests